solana/ts/sdk/ntt.ts

```diff
@@ -86,6 +136,71 @@ export class SolanaNtt<N extends Network, C extends SolanaChains>
     this.pdas = NTT.pdas(this.program.programId);
   }
 
+  async getTransceiver(ix: number): Promise<NttTransceiver<N, C, any> | null> {
+    if (ix !== 0) return null;
+    if (this.whTransceiverAddress === undefined) return null;
+
+    return new SolanaNttWormholeTransceiver(this, new PublicKey(this.whTransceiverAddress));
+  }
+
+  async getMode(): Promise<Ntt.Mode> {
+    const config = await this.getConfig();
+    return config.mode.locking != null ? "locking" : "burning";
+  }
+
+  async isPaused(): Promise<boolean> {
+    const config = await this.getConfig();
+    return config.paused;
+  }
+
+  async *pause(payer: AccountAddress<C>) {
+    const sender = new SolanaAddress(payer).unwrap();
```

Comment:
Same as above re parameter, changing from `AccountAddress<C>` to `AnySolanaAddress` should allow string or PublicKey to be passed, same "unwrap" method can be used to get just a PublicKey out

https://github.com/wormhole-foundation/wormhole-sdk-ts/blob/main/platforms/solana/src/types.ts#L21